src/Regions.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import Regions from './Regions';
+
+test('Regions', () => {
+  const regions = [
+    { id: 1, name: '서울' },
+  ];
+
+  const { getByText } = render(<Regions regions={regions} />);
```

Comment:
```suggestion
  const { getByText } = render((
    <Regions regions={regions} />
  ));
```

이 부분도 코드는 짧지만, 줄바꿈이 들어가주면 읽기 좋을 것 같아요~